README.md

```diff
@@ -47,6 +47,10 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
 * [office-addin-node-debugger](packages/office-addin-node-debugger/README.md)
 
```

Comment:
I think we should add a similar privacy statement here just as Yo Office is doing.  Something like this but more general:

The Yeoman generator for Office Add-ins collects anonymized usage data and sends it to Microsoft. For more details on how we use this data and under what circumstances it may be shared, please see the Microsoft privacy statement.
Examples of data that the Yeoman generator for Office Add-ins collects include:
•   Date, time, and location of project creation
•   The framework that’s selected when the project is created (e.g., React, Angular, etc.)
•   The language that’s selected when the project is created (e.g., TypeScript, JavaScript)
•   The Office application that’s selected when the project is created (e.g., Excel, Word, PowerPoint, Outlook, etc.)
•   Any exceptions or errors that occur when using the Yeoman generator
•   Information about whether the project was created successfully
 
Microsoft uses this data to improve our products, troubleshoot problems, and operate our business.

Reply:
So what exactly would be the specifics then if we are keeping it general, and this would be for Office-Addin Scripts as a whole right?

Reply:
I would just say " Office Addin-Telemetery collects anonymized usage data and sends it to Microsoft. For more details on how we use this data and under what circumstances it may be shared, please see the Microsoft privacy statement. " And the for the bulleted list you can just generally say that the package collects usage data about operations performed and exception call stacks to help diagnose issues.  @akrantz  mentioned that there is documentation we may be able to point to